add-new-exercise.ps1

```diff
@@ -36,30 +36,34 @@ param (
 # Import shared functionality
 . ./shared.ps1
 
-$exerciseName = (Get-Culture).TextInfo.ToTitleCase($Exercise).Replace("-", "")
-$exercisesDir = Resolve-Path "exercises"
-$exerciseDir = Join-Path $exercisesDir $Exercise
-
 function Add-Project {
+    param (
+        [Parameter(Position = 0, Mandatory = $true)][string]$Exercise,
+        [Parameter(Position = 1, Mandatory = $true)][string]$ExerciseName
+    )
+
     Write-Output "Adding project"
 
-    $csProj = "$exerciseDir/$exerciseName.csproj"
+    $exercisesDir = Resolve-Path "exercises"
+    $exerciseDir = Join-Path $exercisesDir $Exercise
```

Comment:
This won't be the correct path, as we have practice and concept exercises. Could you update to:

```suggestion
    $exercisesDir = Resolve-Path "exercises"
    $exerciseDir = Join-Path $exercisesDir "practice" $Exercise
```

Could you add a TODO indicating that we should support concept exercises too?

Reply:
done